build.gradle.kts

```diff
@@ -31,7 +31,9 @@ extra["springAiVersion"] = "1.0.0"
 dependencies {
 
     implementation("org.springframework.ai:spring-ai-starter-mcp-server")
-    implementation("org.apache.solr:solr-solrj:9.8.1")
+    implementation("org.apache.solr:solr-solrj:9.8.1") {
+        exclude(group = "org.apache.httpcomponents")
```

Comment:
BTW I was doing some experimentation on the ability to exclude dependencies.  I saw this could be removed.  In 9.10 (not yet released), this will be more officially supported.  As long as we don't use Http2SolrClient here in this project yet, which has an errant reference to a class that in turn referenced Apache HttpClient.